homeassistant/components/deconz/device_trigger.py

```diff
@@ -209,6 +209,13 @@
     (CONF_DOUBLE_PRESS, CONF_BOTH_BUTTONS): 3004,
 }
 
+AQARA_DOUBLE_WALL_SWITCH_86SW2_MODEL = "lumi.sensor_86sw2"
+AQARA_DOUBLE_WALL_SWITCH_86SW2 = {
```

Comment:
Could we change 86sw2 to the model written on the backside of the device. Now with multiple similar devices I think it is important with clear bookkeeping

Reply:
My Chinese is not so good... 😅

Reply:
Google translate and google give me WXKG02LM?